plugins/quickinspector/quickinspector.cpp

```diff
@@ -228,6 +229,17 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
   return QStringLiteral("Unknown: %1").arg(wrapMode);
 }
 
+static bool qGoodCandidateItem(QQuickItem *item)
```

Comment:
That's because the other functions there are all of the form qTypeToString(), for a set of Qt types. So the "q" prefix for qGoodCandidateItem() indeed doesn't make sense.